### ISSUE TITLE
nm_extract listmode file extracting error message

### ISSUE THREAD

**Issue by Ede1994** [UCL/pet-rd-tools#48]
I try to extract a listMode file from mMR .dcm with ```nm_extract``` and got this error:

```
I1022 16:10:39.149196 14604 NMExtract.cpp:105] Started: Thu Oct 22 16:10:39 2020
I1022 16:10:39.149358 14604 NMExtract.cpp:106] Running 'nm_extract' version: 2.0.0
I1022 16:10:39.149489 14604 Common.hpp:190] Manufacturer: SIEMENS 
I1022 16:10:39.149497 14604 Common.hpp:198] Model name: Biograph_mMR
I1022 16:10:39.149500 14604 MMR.hpp:135] Image type: ORIGINAL\PRIMARY\PET_LISTMODE 
double free or corruption (top)
Cancelled (memory snapshot written)
```

I translated the last line by my own, because I use Linux in German.

**Comment by rijobro** [comment]
From the short error message, I personally haven't got any ideas for what might be wrong. 

Do you have access to any other Siemens mMR listmode .dcm files? If so, does it happen when you try and unlist those?

You could try building in debug mode, even running with a debugger, but these are slightly more involved.

**Comment by KrisThielemans** [comment]
@Ede1994 what version are you using? I get some funny business with the current one, but it does write an error. More info soon

**Comment by KrisThielemans** [comment]
Seems the same problem as #45

**Comment by KrisThielemans** [comment]
Closing as indeed same cause as #45.

Will be fixed with #47
